src/stories/Menu.stories.js

```diff
@@ -38,10 +40,10 @@ storiesOf('Menu', module)
       <ul className="coa-Menu__list">
```

Comment:
Does this mean that MenuList should be a component? and not MenuItem? MenuItem clearly isn't stand-alone. But it seems like the list is? Not sure what makes sense here.

Reply:
Just needed that class so styling would work. MenuList could be a component, but its not yelling out at me to be moved into its own component... yet